bot/code_review_bot/revisions.py

```diff
@@ -61,7 +61,7 @@ def publish(self, days_ttl=30):
         ), "Only publish on online Taskcluster tasks"
         self.url = taskcluster.upload_artifact(
             "public/patch/{}".format(self.name),
-            self.content,
+            str(self.content, "utf-8"),
```

Comment:
This is going to do the decoding, we should do the opposite (`self.content` is a string and we want bytes)